tools/scylla-sstable.cc

```diff
@@ -2090,6 +2090,12 @@ void upgrade_operation(schema_ptr schema, reader_permit permit, const std::vecto
     }
 }
 
+void dump_schema_operation(schema_ptr schema, reader_permit permit, const std::vector<sstables::shared_sstable>& sstables,
+        sstables::sstables_manager& sst_man, const bpo::variables_map& vm) {
```

Comment:
@denesb , this likely needs rebase. #27021 added `const db::config&` argument to operation callbacks and this one doesn't have it. No patch-level conflicts, but compilation will fail

Reply:
Ok, will rebase and verify that it builds.

Reply:
While at it, I will also add a test.

Reply:
Done in v2.